install_bootstrap

```diff
@@ -58,14 +58,14 @@ function generate_password {
 # $4 - reference to the user's answer.
 function ask_question()
 {
-  question_str=$1
-  prev_val=$2
-  val_name=$3
-  answer=''
+    question_str=$1
```

Comment:
Did you change tabs to spaces or vice-versa, or otherwise do some reformatting?

With exceptions, it's generally best not to add a feature and then also reformat a bunch of the file; it makes it harder to tell what changed.

This comment applies to the whole file, not just this line.

OTOH, if part of your change was just to reformat the indentation, then maybe it's OK.

Reply:
I did all of the reformatting in a separate commit https://github.com/DataBiosphere/cgp-deployment/pull/56/commits/20043e98f3e7306450fd98558d2a051671a8d7e5, that way you would be able to tell the changes apart. The indentation was very inconsistent and it was almost necessary to adjust before adding my own code.